grafana-plugin/src/containers/MobileAppVerification/parts/DisconnectButton/DisconnectButton.tsx

```diff
@@ -1,17 +1,28 @@
 import React, { FC } from 'react';
 
 import { Button } from '@grafana/ui';
+import cn from 'classnames/bind';
 
 import WithConfirm from 'components/WithConfirm/WithConfirm';
 
+import styles from './DisconnectButton.module.scss';
+
+const cx = cn.bind(styles);
+
 type Props = {
+  qrCode?: string;
```

Comment:
is this prop needed?

Reply:
Forgotten, I wanted initially to pass the real qr code instead of a dummy png.